webapp-django/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -143,19 +143,17 @@ def show_bug_link(bug_id):
 
 
 @library.global_function
-def bugzilla_submit_url(raw_crash, report, parsed_dump, crashing_thread, bug_product):
-    url = "https://bugzilla.mozilla.org/enter_bug.cgi"
-
+def generate_create_bug_url(template, raw_crash, report, parsed_dump, crashing_thread):
     # Some crashes has the `os_name` but it's null so we
     # fall back on an empty string on it instead. That way the various
     # `.startswith(...)` things we do don't raise an AttributeError.
     op_sys = report.get("os_pretty_version") or report["os_name"] or ""
 
-    # At the time of writing, these pretty versions of the OS name
-    # don't perfectly fit with the drop-down choices that Bugzilla
-    # has in its OS drop-down. So we have to make some adjustments.
+    # At the time of writing, these pretty versions of the OS name don't perfectly fit
+    # with the drop-down choices that Bugzilla has in its OS drop-down. So we have to
+    # make some adjustments.
     if op_sys.startswith("OS X "):
-        op_sys = "Mac OS X"
+        op_sys = "macOS"
```

Comment:
When working on this, I discovered that this op sys had changed in Bugzilla--there isn't a "Mac OS X" anymore. So I fixed it.

The rep_platform is wrong, however. I'll write up a bug for that later.

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1659961